components/AccessibilityModal.tsx

```
@@ -0,0 +1,50 @@
+  visible: boolean;
+  onClose: () => void;
+};
+
+const AccessibilityModal: React.FC<AccessibilityModalProps> = ({ accessibility, visible, onClose }) => {
+  return (
+    <ListingModal title="Services et accessibilité" visible={visible} onClose={onClose}>
+      <ScrollView>
+        {Object.entries(accessibility).map(([key, value], index) => (
+          <View key={key} style={styles.accessibilityItem}>
+            <Text style={styles.accessibilityLabel}>{translateAccessibilityKeys(key)}</Text>
+            <Text style={styles.accessibilityValue}>{!!value ? 'Oui' : 'Non'}</Text>
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)